cmd/podman/ps.go

```diff
@@ -158,10 +158,9 @@ var (
 	psCommand     cliconfig.PsValues
 	psDescription = "Prints out information about the containers"
 	_psCommand    = &cobra.Command{
-		Use:     "list",
-		Aliases: []string{"ls", "ps"},
```

Comment:
Did you mean to remove 'ls' and 'list' aliases?

Reply:
yes, per @rhatdan